frontend/src/Onboarding/TransferableCreditScreen.tsx

```diff
@@ -218,7 +218,7 @@ const TransferableCreditScreen: React.FC = () => {
         );
       }
 
-      createPlanForUser(userId!, token, {
+      createPlanForUser(userId!, {
```

Comment:
Can we also remove where we define token in this file? (Unless we use it elsewhere)

Reply:
Yea it's used in another call. I'll put up a separate PR after this one to clean it all up